pkg/utils/has/controller.go

```diff
@@ -527,7 +527,7 @@ func (h *SuiteController) WaitForComponentPipelineToBeFinished(component *appser
 		if err != nil {
 			GinkgoWriter.Printf("attempt %d/%d: PipelineRun %q failed: %+v", attempts, maxRetries+1, pr.GetName(), err)
 			// Retry the PipelineRun only in case we hit the known issue https://issues.redhat.com/browse/SRVKP-2749
-			if attempts == maxRetries+1 || pr.GetStatusCondition().GetCondition(apis.ConditionSucceeded).GetReason() != "CouldntGetTask" {
+			if attempts == maxRetries+1 || pr.GetStatusCondition().GetCondition(apis.ConditionSucceeded).GetReason() != "CouldntGetTask" || pr.GetStatusCondition().GetCondition(apis.ConditionSucceeded).GetReason() != "PipelineValidationFailed" {
```

Comment:
should we really retry in this case? according to [this bug](https://issues.redhat.com/browse/SRVKP-2880) it seems that `PipelineValidationFailed` means in fact failed pipeline.